### PR TITLE
Update delete post be

### PR DESCRIPTION
**Pull request by rostyk1331**

### CODE REVIEW

File: server/src/helpers/getTags.post.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const getTags = (description: string): any => {
```
Comment (InnaKutsik) [inline server/src/helpers/getTags.post.ts]: any =====> string[]

File: server/src/controllers/post.controllers/update.post.by.id.ts [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
 
-export const update = (req: Request, res: Response, next: NextFunction): void => {
-    // TODO it will be implemented later
+export const update = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+        const id = req.params.id;
+        const description = req.body.description;
+        const tags = getTags(description);
+
+        const updPost = await updatePost(id, description, tags, next);
```
Comment (InnaKutsik) [inline server/src/controllers/post.controllers/update.post.by.id.ts]: just on bd or img too?

File: server/src/helpers/getTags.post.ts (hunk 1 above)
Comment (rostyk1331) [reply server/src/helpers/getTags.post.ts]: Fixed.

File: server/src/controllers/post.controllers/update.post.by.id.ts (hunk 2 above)
Comment (rostyk1331) [reply server/src/controllers/post.controllers/update.post.by.id.ts]: We decided, that user can only update a description of post, but not an image. The same functionality has Instagram. Or should we allow updating img too?

Comment (InnaKutsik) [reply server/src/controllers/post.controllers/update.post.by.id.ts]: Ok, that your decision😉

**Pull request merged**